src/transformers/models/owlvit/feature_extraction_owlvit.py

```diff
@@ -143,6 +183,78 @@ def post_process(self, outputs, target_sizes):
 
         return results
 
+    def post_process_one_shot_object_detection(self, outputs, threshold=0.6, nms_threshold=0.3, target_sizes=None):
+        """
+        Converts the output of [`OwlViTForObjectDetection.image_guided_detection`] into the format expected by the COCO
+        api.
+
+        Args:
+            outputs ([`OwlViTImageGuidedObjectDetectionOutput`]):
+                Raw outputs of the model.
+            threshold (`float`, *optional*, defaults to 0.6):
+                Minimum confidence threshold to use to filter out predicted boxes.
+            nms_threshold (`float`, *optional*, defaults to 0.3):
```

Comment:
Just a question, is NMS only applied for image-guided object detection?

Reply:
Yes but it can be applied to text-guided object detection too, I will add an improved `post_process_object_detection` method in a separate PR.